solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/OriginalScoreFeature.java

```diff
@@ -88,8 +85,7 @@ public float score() throws IOException {
         // This is done to improve the speed of feature extraction. Since this
         // was already scored in step 1
         // we shouldn't need to calc original score again.
-        return hasDocParam(ORIGINAL_DOC_SCORE) ? (Float) getDocParam(ORIGINAL_DOC_SCORE)
-            : originalScorer.score();
+        return getOriginalDocScoreOrDefault(originalScorer.score());
```

Comment:
This defeats the purpose of what we were doing before.  We looked up the original score in the params and used it, and only if it didn't exist did we go ahead and re-score the original query